src/components/pages/Home.vue

```diff
@@ -4,6 +4,26 @@
 			<img class="logo" src="static/klasa.svg">
 			<br>
 			<br>
+			<div class="unmaintainedBanner has-text-left">
+				<h2 class="title is-3">
+					<strong>
+						Klasa is not maintained anymore; here's why:
+					</strong>
+				</h2>
+				Due to the differences between us, the community, and one of the ex Core Developers,
+				Dirigeants and all its projects are now unable to be maintained, as most of the active
+				contributors and Core Developers were banned from the Discord server.
+				<br>
+				We recommend you look for another framework to use. You may still use Klasa as is,
+				but you're doing so at your own risk, and <b>we wil not provide support or bugfixes</b>.
+				<br>
+				We'd like to thank everyone who contributed to our projects, and our community!
+				It's sad to see this come to an end as it stands, however our hands are tied.
+				<br>
+				<br>
+				If you wish to learn more about this incident, we invite you to read its writeup, as recorded by
+				Favna, <a href="https://gist.github.com/Favna/b9b6c23b6e545f7eddfdb55027a182df"> found here</a>
```

Comment:
```suggestion
				Favna#0001 (one of the more active members of Dirigeants before and during the events), <a href="https://gist.github.com/Favna/b9b6c23b6e545f7eddfdb55027a182df"> found here</a>
```

Reply:
no:heart:

Reply:
dude
I put that in there because most people won't know me
they will be like "who the fuck is this Favna claiming he knows what happened"
I didn't put that in there to gloat at all